home-page.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Beyond the Reel</title>
+
+    <!--Will add an icon-->
+    <link rel="shortcut icon" href="" type="" />
+  </head>
+
+  <header>
+    <nav>
+      <section>
+        <a href="/about-us.html">About Us</a><br />
+        <a href="/contact.php">Contact</a><br />
+        <a href="/news.html">News</a><br />
+        <a href="/archive.html">Video Archive</a><br />
+      </section>
+    </nav>
+  </header>
+
+  <body>
+    <div class=""></div>
+
+    <div></div>
```

Comment:
If these are placeholders you should add a comment regarding their use

Reply:
This has been identified as a container div.